PKGBUILD

```diff
@@ -57,21 +187,22 @@ pkgver() {
 
 prepare() {
   cd $pkgname-$pkgver
-
-  # use real directories for examples and profiles, as symlinks do not work
-  rm -fv $pkgname/{examples,profiles}
 }
 
 build() {
   cd $pkgname-$pkgver
-
   python -m build --wheel --no-isolation
   PYTHONDONTWRITEBYTECODE=1 make man -C docs
 }
 
 package() {
-  cd "$pkgname-$pkgver"
-
+  cd $pkgname-$pkgver
   python -m installer --destdir="$pkgdir" dist/*.whl
   install -vDm 644 docs/_build/man/archinstall.1 -t "$pkgdir/usr/share/man/man1/"
 }
+
+check() {
```

Comment:
If `check()` does nothing, it should not be here.
Once it calls something, it should be ordered before `package()` as that is when it is called.

Reply:
To not forget that we *want* tests, is it fine if I comment it out instead of removing it?